config/locales/en/images/edit.yml

```diff
@@ -12,5 +12,8 @@ en:
         requirements: You need to
       guidance:
         alt_text_govspeak: A simple and specific description of what the image shows. This helps screen-readers and search engines.
-        caption_govspeak: This text appears on the page under the image.
-        credit_govspeak: You must have the rights to use this image. Images should be credited to their source. For example, Getty Images. Open Government Licence (OGL) images do not need to be credited.
+        caption_govspeak: Optional. A description of what the image shows. Caption appears next to the image.
+        credit_govspeak: |
+          Images licensed from a 3rd party must be credited. Government images do not need a credit.
+
+          [Full image credit guidance](https://www.gov.uk/guidance/content-design/image-copyright-standards-for-gov-uk)
```

Comment:
We should open this with `target="_blank"` as otherwise clicking it in through the modal view will leave the edit.

Reply:
I agree. But I don't know how to do that with markdown. Can you show me?

Reply:
`[Full image credit guidance](https://www.gov.uk/guidance/content-design/image-copyright-standards-for-gov-uk){:target="_blank"}`

Reply:
☝️ the moment when markdown loses its scope